test/extended/images/helper.go

```diff
@@ -16,3 +24,48 @@ func GetImageLabels(c client.ImageStreamImageInterface, imageRepoName, imageRef
 	}
 	return image.Image.DockerImageMetadata.Config.Labels, nil
 }
+
+// ModifySourceCode will modify source code in the pod of the application
+// according to the sed script.
+func ModifySourceCode(oc *exutil.CLI, selector labels.Selector, sedScript, file string) error {
```

Comment:
I've looked at it again and it uses the WaitForPods() function which takes a label, the point being that the function will also make sure the pod is running. I could move the waiting-until-pod-is-running logic outside of that function, but I think it will only make the tests look less comprehensible.